src/main/java/com/eternalcode/discordapp/leveling/command/Top10LevelCommand.java

```diff
@@ -0,0 +1,50 @@
+package com.eternalcode.discordapp.leveling.command;
+
+import com.eternalcode.discordapp.leveling.Level;
+import com.eternalcode.discordapp.leveling.LevelService;
+import com.jagrosh.jdautilities.command.SlashCommand;
+import com.jagrosh.jdautilities.command.SlashCommandEvent;
+import net.dv8tion.jda.api.EmbedBuilder;
+import panda.utilities.text.Formatter;
+
+import java.util.List;
+
+public class Top10LevelCommand extends SlashCommand {
```

Comment:
Ja bym to w sumie zgeneralizował, może na 1 stronie faktycznie byłoby top 10, a potem można byłoby za pomocą buttonów to przewijać jakoś